### ISSUE TITLE
Activity Browser poster design and contents: notice & request for input

### ISSUE THREAD

**Issue by TMillross** [LCA-ActivityBrowser/activity-browser#149]
Hi folks. As mentioned on Slack, I'm developing a poster on the AB project that will be presented later this month. Under the assumption that most conference participants will not be too comfortable with the technical/code details, the emphasis will be on the relevance and environment of the system. The current thinking for design (under iterative improvement) is: 
* Title at top
* History/timeline of project along the bottom: Past to future plans/vision (present is the rest of poster)
* Rest of poster organised around an extended version of a '[Context View](https://www.viewpoints-and-perspectives.info/home/viewpoints/context/)' diagram ([example of one I previously helped make](https://delftswa.gitbooks.io/desosa-2017/content/neovim/images-team-neovim/context-model.png)). 

**Extended Context View Layout**
AB and BW2 will be in the middle, with their relationship defined and pony/sausage mascots nearby :D Fine arrows will connect to objects of the project's context as preliminarily described below. The more significant sections that should be explained/shown in detail will be further away from the centre in their own sections. For example:  AB --has functionality--> [Software features poster section with screenshots and explanations etc.]

Source | Relationship | Destination
-- | -- | --
**Environment** |   |  
both | Are available to | Everyone with a computer
both | Costs | Free
both | Run on | Windows, MacOS, Linux
both | Exist to support | LCA researchers and advanced practitioners
both | Part of IE trend toward | **IE Open Science & Data** section of poster. Open and  collaboratively developed methods and data: high quality science. Reference  IndEcol Github, related project, some key publications, community members etc.
both | Provides functionality related to | SimaPro, CMLCA,OpenLCA, GaBi
  |   |  
**Overview** |   |  
AB | Provides GUI into | Brightway2 (in middle of poster)
BW | Performs | Advanced Lifecycle Assessment
Lifecycle Assessment | Quantifies | Lifecycle environmental impacts of products
BW | User interaction via | Keyboard through Python scripts
AB | User interaction via | Mouse and keyboard through GUI
AB | Displays results through | Tables charts and diagrams using Seaborn, MatplotLib and Floweaver Sankey diagrams
BW | Provides data migrations from | ecospold1, ecospold1-lcia, ecospold2, excel, exiobase, simapro CSV, and   simapro CSV-lcia
BW | Has the mascot | Sausage (name?)
AB | Has the mascot | Pony (name?)
  |   |  
**Development** |   |  
both | Programmed using | Python (maybe list some libs)
AB | Programmed using | PyQT GUI Toolkit
BW | Version control and project managed on | BitBucket, Mercurial, [Groups.io](https://groups.io/g/brightway-dev), Sphinx
AB | Version control and project managed on | GitHub, Git, Slack
AB | Tested, synced and deployed by | PyTest, Travis CI, Coveralls
BW | Tested, synced and deployed by | PyTest, Appveyor
AB | Developed by | Bernard Steubing, Adrian Haas, Giuseppe Cardellini, Chris Mutel, (CML Students:) Antoine   Coudard, Anje Knottnerus, Marc van der Meide, Thomas Millross,  Max Reinicke, Maarten Schut
BW | Developed by | Chris Mutel, Adrian Haas         Giuseppe Cardellini         Andreas Froemelt         Niko Heeren         Aurelian Jaggi         Maghimai Marcus         Marie de Saxcé         Low Kian Seong         Bernhard Steubing         Pascal Lesage         Manuel Klarmann         Tomas Navarrete         Guillaume Audard
BW | Licensed under | BSD-3 (permissive free software license)
AB | Licensed under | GNU General Public License
  |   |  
**Functionality** |   |  
AB | Supports the graphical functionality | Functionality section of poster, including data management, results,   supply chain traversal etc. with screenshots
BW | Supports the LCA functionality | Highlight some key features relevant to the conference audience (e.g.   uncertainty, allocation)

* The relationships that are listed as different for AB and BW rather than 'both' (e.g. License) will be colour-coded or similar for effortless discernment by the viewer.
* The maximum dimensions of the poster are 4ft by 4ft but I'll likely opt for around 4ft by 3ft (122 x 91cm). So lots of visuals and information could be included. But to avoid clutter, I'll delete the less significant context relationships as necessary. 

__If there's anything you think I should add or mistakes in the detail: please comment here or on Slack!__

**Comment by TMillross** [comment]
Thanks @haasad, I will indeed!

**Comment by TMillross** [comment]
The poster draft is uploaded to Slack awaiting feedback on any final changes before printing on Wednesday!  Thanks in advance, @haasad @cmutel @cardosan @bsteubing

**Comment by TMillross** [comment]
This was the poster submitted and displayed at the conference. Thanks for those that contributed! [The LCA Activity Browser Graphical Interface for Brightway2_V1.pdf](https://github.com/LCA-ActivityBrowser/activity-browser/files/2090377/The.LCA.Activity.Browser.Graphical.Interface.for.Brightway2_V1.pdf)
We received some great feedback and encouragement during the session. A number of professors teaching LCA at Bachelors, Masters or Professional level were present. They represented Santa Barbara, Toronto & Michigan (and some others who's affiliations I did not record). They were broadly keen to use AB for their courses in the coming years. We highlighted the ongoing development as a caveat, but otherwise gave information and encouragement.
Since returning from the conference I've not found time for this project. But having cleared two other pieces of work now, intend to commit plenty of code in the next 2 or 3 weeks.
